controllers/noobaa-rules.yaml

```diff
@@ -1,104 +1,96 @@
          labels:
            origin: data-federation-service
-         record: noobaa_num_buckets_claims
+         record: namespace:noobaa_buckets_claims:max
        - expr: |
-           NooBaa_num_unhealthy_bucket_claims{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_unhealthy_bucket_claims{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_num_unhealthy_bucket_claims
+         record: namespace:noobaa_unhealthy_bucket_claims:max
        - expr: |
-           NooBaa_num_unhealthy_namespace_buckets{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_unhealthy_namespace_buckets{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: job:noobaa_total_unhealthy_namespace_buckets
+         record: namespace:noobaa_unhealthy_namespace_buckets:max
        - expr: |
-           NooBaa_num_namespace_buckets{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_accounts_num{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: job:noobaa_namespace_bucket_count
+         record: namespace:noobaa_accounts:max
        - expr: |
-           NooBaa_accounts_num{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_total_usage{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_accounts_num
+         record: namespace:noobaa_usage:max
        - expr: |
-           NooBaa_total_usage{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_unhealthy_namespace_resources{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_total_usage
+         record: namespace:noobaa_unhealthy_namespace_resources:max
        - expr: |
-            NooBaa_num_unhealthy_namespace_resources{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_namespace_resources{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_num_unhealthy_namespace_resources
+         record: namespace:noobaa_namespace_resources:max
        - expr: |
-           NooBaa_num_namespace_resources{namespace="redhat-data-federation"}
+           max without(instance) (NooBaa_num_namespace_buckets{namespace="redhat-data-federation"})
          labels:
            origin: data-federation-service
-         record: noobaa_num_namespace_resources
+         record: namespace:noobaa_namespace_buckets:max
        - expr: |
-           NooBaa_namespace_resource_status{namespace="redhat-data-federation"}
+           count(NooBaa_namespace_bucket_status{namespace="redhat-data-federation"} == 0)
```

Comment:
the recording rules won't use the same labelset. E.g. `namespace:unhealthy_noobaa_namespace_buckets:count` will have only the `origin` label while `namespace:noobaa_namespace_buckets:max` will have all labels except instance. To avoid future complications when doing arithmetic on both metrics (e.g. "namespace:unhealthy_noobaa_namespace_buckets:count / namespace:noobaa_namespace_buckets:max" will always return no data), it's better to be consistent. 

So either modify this one to be:

```
count without(instance, bucket_name) (NooBaa_namespace_bucket_status{namespace="redhat-data-federation"} == 0)
```

or modify namespace:noobaa_namespace_buckets:max to be:

```
max(NooBaa_num_namespace_buckets{namespace="redhat-data-federation"})
```

As I said earlier, the second form is probably less error-prone because you don't have to enumerate all labels that need to be dropped.